docs/apu_flashing_with_rpi.md

```diff
@@ -23,7 +23,7 @@ time to connect RPi to apu.
 | SPI0 MISO  | 21             | 5 SPIDI                 |
 | SPI0 MOSI  | 19             | 6 SPIDO                 |
 
-> Except that before flashing we have to make sure that APU2 is on and
+> Except that before flashing we have to make sure that APUx is on and
   in S5 state. That means we have to connect pins 2 and 3 on J2 header
```

Comment:
@adamwawrzynski3mdeb w tej notatce jest wzmianka o zwieraniu pinów na headerze J2. Poszukaj na schematach czy to aby napewno na każdej platformie jest J2. Schematy są u Nas na cloud i na stronei pcengines.ch

Reply:
@miczyg1 Sprawdziłem, na każdej platformie są identyczne.